src/main/python/calculate_hashtables/main.py

```diff
@@ -0,0 +1,61 @@
+"""
+Calculate number of hashtabels and size of a band
+using the threshold and sample size of wmh
+"""
```

Comment:
If this code is similar or comes from another codebase, could you please add a permlink URL in here as well?

I.e https://github.com/src-d/apollo/blob/bbb9f83fffc93e791fa59f064efadefa270f400a/apollo/hasher.py#L20

Reply:
As this code seems to come from [MIT-licensed project](https://github.com/ekzhu/datasketch/blob/master/LICENSE), so you also need to update https://github.com/src-d/gemini/blob/master/LICENSE#L677 by adding a new [section for MIT](https://github.com/apache/spark/blob/master/LICENSE#L276) and then as before - add a link to both, original project and a path to `main.py` in our codebase that contains this code.